events/app.py

```diff
@@ -53,8 +53,7 @@ def add_event():
 
 def build_event_info(info, time):
     """Adds created_at time to event info dict."""
-    info['created_at'] = time
-    return info
+    return dict(created_at=time, **info)
```

Comment:
`info` comes from `request.form` which is user supplied, if I'm correct. How are you handling the case where `info` already has a `created_at` attribute? I think this dict constructor might raise a `KeyError` in that case.

Reply:
Actually, you get a `TypeError`, surprisingly.

I just discovered [PEP 448](https://www.python.org/dev/peps/pep-0448/), which allows this form:

```python
return {**info, 'created_at': time}
```

Reply:
Done in [d1ade21](https://github.com/knative-portability/large-events/pull/4/commits/d1ade21683ef7f0d5d8c52a230d650691c86cd9f)